example-playbooks/replace_node/replace_node.yml

```diff
@@ -24,6 +24,30 @@
       when: hostvars[new_node]['broadcast_address'] in scylla_seeds or new_node in scylla_seeds
       run_once: true
 
+    - name: Stop scylla-server on the node being replaced
+      block:
+        - name: run nodetool drain
+          shell: |
+            nodetool drain
```

Comment:
`nodetool drain` is blocking, so you would need to run it using `async_task`.
However why do you even drain this node? You are about to kill it!

Reply:
Fixed